arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -1228,26 +1382,24 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "The collection of volumes.",
-            "schema": {
-              "$ref": "#/definitions/VolumeList"
-            }
+          "202": {
+            "description": "Accepted the request to backup the backup policy."
+          },
+          "204": {
+            "description": "Successfully completed backup of the backup policy."
           }
         },
         "deprecated": false,
-        "x-ms-pageable": {
-          "nextLinkName": null
-        }
+        "x-ms-long-running-operation": true
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}/devices/{deviceName}/volumeContainers/{volumeContainerName}/volumes/{volumeName}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/managers/{managerName}/devices/{deviceName}/backupPolicies/{backupPolicyName}/schedules": {
       "get": {
         "tags": [
-          "Volumes"
+          "BackupSchedules"
         ],
-        "description": "Returns the properties of the specified volume name.",
-        "operationId": "Volumes_Get",
+        "description": "Gets all the backup schedules in a backup policy.",
+        "operationId": "BackupSchedules_ListByBackupPolicy",
```

Comment:
We have schedules in backup-policies as well as in BandwidthSetting. To distinguish between the two, this is intentionally named as BackupSchedule.

Only tracked resource in our APIs is 'manager'. All others including 'BackupSchedule' are non-tracked resources.